src/thunderbots/software/ai/hl/stp/evaluation/baller.h

```diff
@@ -0,0 +1,19 @@
+#include "ai/world/ball.h"
+#include "ai/world/field.h"
+#include "ai/world/team.h"
+
```

Comment:
Similar comment to function name below, can we re-name this file? Maybe `possession.h`?

Reply:
renamed